MoviesMVVM/MoviesMVVM/AppDelegate.swift

```diff
@@ -29,4 +30,28 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
         _ application: UIApplication,
         didDiscardSceneSessions sceneSessionsSet: Set<UISceneSession>
     ) {}
+
+    lazy var persistentContainer: NSPersistentContainer = {
+        let container = NSPersistentContainer(name: "MovieEntity")
+        container.loadPersistentStores(completionHandler: { storeDescription, error in
+            if let error = error as NSError? {
+                fatalError("Unresolved error \(error), \(error.userInfo)")
```

Comment:
```suggestion
                fatalError("Unresolved error \(error), \(error.userInfo)")
```
тоже можно в константу